File: jmlparser-jml-tools/src/main/java/com/github/jmlparser/lint/rules/LocationSetValidator.java
package com.github.jmlparser.lint.rules;

import com.github.jmlparser.lint.LintRuleVisitor;

/**
 * @author Alexander Weigl
 * @version 1 (12/29/21)
 */
public class LocationSetValidator extends LintRuleVisitor {
    public static final String ASSIGNABLE_ARRAY_ONLY = "You can only use '[*]' on arrays";
    public static final String ASSIGNABLE_CLASS_ONLY = "You can only use '.*' on classes and interfaces";
}

File: tools/lint/src/main/kotlin/io/github/jmltoolkit/lint/rules/LocationSetValidator.kt
package io.github.jmltoolkit.lint.rules

import io.github.jmltoolkit.lint.LintRuleVisitor


/**
 * @author Alexander Weigl
 * @version 1 (12/29/21)
 */
object LocationSetValidator : LintRuleVisitor() {
    const val ASSIGNABLE_ARRAY_ONLY: String = "You can only use '[*]' on arrays"
    const val ASSIGNABLE_CLASS_ONLY: String = "You can only use '.*' on classes and interfaces"
}
